src/main/resources/META-INF/rewrite/java-version-17.yml

```
@@ -47,3 +49,41 @@ recipeList:
       version: 17
   - org.openrewrite.java.migrate.maven.UseMavenCompilerPluginReleaseConfiguration:
       releaseVersion: 17
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.DeprecatedJavaxSecurityCert
+displayName: Use javax.security.cert
+description: The javax.security.cert package has been deprecated for removal
+tags:
+  - java17
+recipeList:
+  - org.openrewrite.java.ChangeType:
+      oldFullyQualifiedTypeName: javax.security.cert
+      newFullyQualifiedTypeName: java.security.cert
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.RemovedLegacySunJSSEProviderName
+displayName: Use "SunJSSE"
+description: The com.sun.net.ssl.internal.ssl.Provider provider name was removed
```

Descriptions can use backticks and other markdown symbols to enhance the readability in the docs; best to apply those around code references where we can.
```suggestion
description: The `com.sun.net.ssl.internal.ssl.Provider` provider name was removed
```